docker_deploy/Dockerfile-ui

```diff
@@ -0,0 +1,22 @@
+FROM node:4.8-wheezy
+
+# Install Node.js dependencies
+COPY package.json /
+RUN npm install
+
+# Copy project files to docker container
+COPY views/ /views/
+COPY ui.js /
+
+CMD node /ui.js
+
+#
+# NOTES
+#
+# Build this container with (from project root dir):
+# docker build -t mike/osv-microservice-demo . -f docker_deploy/Dockerfile-ui
```

Comment:
rename to `-t mike/osv-microservice-demo-ui` as this only uploads UI-related scripts and files.